sdk/storage/azure-storage-blob/tests/test_retry_async.py

```diff
@@ -471,4 +474,38 @@ async def test_invalid_account_key_async(self, storage_account_name, storage_acc
         # No retry should be performed since the signing error is fatal
         self.assertEqual(retry_counter.count, 0)
 
+    @staticmethod
+    def _count_wrapper(counter, func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            counter[0] += 1
+            return func(*args, **kwargs)
+        return wrapper
+
+    @BlobPreparer()
+    @AsyncStorageTestCase.await_prepared_test
+    async def test_streaming_retry_async(self, storage_account_name, storage_account_key):
```

Comment:
If the recording is giving a lot of trouble, we can just mark this live only as you suggested.

Reply:
marked as live-onlu